static/app/components/ai/SeerIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import {forwardRef} from 'react';
+
+import type {SVGIconProps} from 'sentry/icons/svgIcon';
+import {SvgIcon} from 'sentry/icons/svgIcon';
+
+const SeerIcon = forwardRef<SVGSVGElement, SVGIconProps>((props, ref) => {
+  return (
+    <SvgIcon {...props} ref={ref} viewBox="0 0 30 30">
+      <defs>
+        <style>
+          {`.cls-1 {
+            fill: none;
+            stroke: currentColor;
+            stroke-linecap: round;
+            stroke-linejoin: round;
+            stroke-width: 2.25px;
+          }`}
+        </style>
```

Comment:
we don't typically use style like this inside svg icons. should apply it using emotion to a styled('path')